README.md

```diff
@@ -0,0 +1,30 @@
+# lambda-update-image-builder-pipeline
+Template repository for terraform modules. Good for any cloud and any provider.  
```

Comment:
I guess update the subtitle and stuff in a subsequent PR?

Reply:
ugh...yeah I'll update it in the one that actually adds the terraform stuff. Gonna rename the repo too. Originally it was gonna be 2 separate repos, but I like it better this way